### PR TITLE
Moved Jobs navigation to its own parent group

### PR DESCRIPTION
**Pull request by briddo**
Fixes: #880

### CODE REVIEW

File: nautobot/extras/navigation.py [hunk 1]
```diff
@@ -96,8 +96,57 @@
         ),
     ),
     NavMenuTab(
-        name="Extensibility",
+        name="Jobs",
         weight=800,
+        groups=(
+            NavMenuGroup(
+                name="Jobs",
+                weight=400,
```
Comment (glennmatthews) [inline nautobot/extras/navigation.py:104]: Can probably reset this weight to 100.

**Review by glennmatthews — COMMENTED** [review]
Thanks for the PR! Would you mind attaching a screenshot of the updated menu bar? I'm trying to decide whether I like the positioning of `Jobs` between `Secrets` and `Extensibility`. :-)

File: nautobot/extras/navigation.py (hunk 1 above)
Comment (briddo) [reply nautobot/extras/navigation.py]: Done. Well spotted!

**Review by glennmatthews — APPROVED** [review]

**Comment by briddo** [comment]
@glennmatthews yeah sure here it is.
Screenshot 2021-12-21 at 16 12 05
.

**Pull request merged**